src/search/landmarks/landmark_count_heuristic.h

```diff
@@ -23,11 +23,21 @@ class LandmarkCountHeuristic : public Heuristic {
     const bool conditional_effects_supported;
     const bool admissible;
     const bool dead_ends_reliable;
+    const int derived_landmark_cost;
 
     std::unique_ptr<LandmarkStatusManager> lm_status_manager;
     std::unique_ptr<LandmarkCostAssignment> lm_cost_assignment;
     std::unique_ptr<successor_generator::SuccessorGenerator> successor_generator;
 
+    /*
+      TODO: The following vectors are only used in the non-admissible case. This
+       is bad design, but given this is already the case for
+       *lm_cost_assignment* above, this is just another indicator to change this
+       in the future.
+    */
```

Comment:
Agreed, we should split the admissible and the inadmissible landmark heuristics in a future issue.